static/app/utils/performance/anomalies/anomaliesQuery.tsx

```diff
@@ -46,7 +46,6 @@ export type AnomalyPayload = {
 
 function transformStatsTimes(stats: AnomalyStatsData) {
   stats.data.forEach(d => (d[0] = d[0] * 1000));
-  stats.data = stats.data.slice((stats.data.length * 4) / 6, stats.data.length);
```

Comment:
something has gone wrong

Reply:
looks like my recent commit got caught in a rebase 🤷